cmd/sync/sync.go

```diff
@@ -23,13 +27,39 @@ import (
 
 type syncFlags struct {
 	// project files polling interval
-	interval time.Duration
-	full     bool
-	watch    bool
-	output   flags.Output
-	exclude  []string
-	include  []string
-	dryRun   bool
+	interval    time.Duration
+	full        bool
+	watch       bool
+	output      flags.Output
+	exclude     []string
+	include     []string
+	dryRun      bool
+	excludeFrom string
```

Comment:
So you can only specify this option once? rsync supports multiple, let's make this a slice as well?